docs/dev-guide/contributing/branching.rst

```diff
@@ -88,16 +95,20 @@ Choosing an Upstream Branch
 ---------------------------
 
 When creating a bug fix or feature branch, it is very important to choose the
-right upstream branch. The general rule is to always choose the oldest upstream
-branch that will need to contain your work.
+right upstream branch. The general rule is to always choose the oldest supported upstream
+branch that will need to contain your work. For more info see above
+:ref:`Feature Branches <feature_branches>` or :ref:`Bug Fix Branches <bug_fix_branches>`.
 
 .. _commit_messages:
 
 Commit Messages
 ---------------
 
 The primary commit in a bug fix should have a log message that starts with
-'<bug_id> - ', for example ``123456 - fixes a silly bug``.
+'<bug_id> - ', for example ``123456 - fixes a silly bug`` and in case of
```

Comment:
Can we reword this? The 'starts with' part confused me when I was doing my first commit. Maybe something like:

The commit message should always be in the following format:

A description of the changes
closes #bug_id
link to bug